jenkins/ci.suse.de/ccp.yaml

```diff
@@ -0,0 +1,66 @@
+- job:
+    name: 'ccp-full-periodic'
```

Comment:
job name should start with `cloud-` .we share the jenkins server with others.

Reply:
Done.